lib/Jsrt/ChakraCore.h

```diff
@@ -163,7 +163,7 @@ JsGetModuleHostInfo(
     _In_ JsModuleHostInfoKind moduleHostInfo,
     _Outptr_result_maybenull_ void** hostInfo);
 
-#ifndef NTBUILD
+#ifdef CHAKRACOREBUILD_
```

Comment:
The styling was @curtisman 's choice and I also liked it. The `_` at the end gives `private use only` impression.